lib/experimental/class-wp-menu-conversion-gutenberg.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * WP_Menu_Conversion_Gutenberg class
+ *
+ * @package gutenberg
+ * @since 6.3.0
+ */
+
+/**
+ * Manages Fallback behavior for Navigation menus.
+ *
+ * @access public
+ */
+class WP_Menu_Conversion_Gutenberg {
+
+	/**
+	 * The Menu term object of the menu to convert.
+	 *
+	 * @var WP_Term
+	 */
+	private $menu;
+
+	/**
+	 * Constructor
+	 *
+	 * @param WP_Term $menu The Menu term object of the menu to convert.
+	 */
+	public function __construct( $menu ) {
+		$this->menu = $menu;
+	}
+
+	/**
+	 * Converts a Classic Menu to blocks.
+	 *
+	 * @return array the serialized and normalized parsed blocks.
+	 */
+	public function convert() {
+		// BEGIN: Code that already exists in wp_nav_menu().
```

Comment:
This comment is from the original code but I have been unable to track down the code it supposedly references in `wp_nav_menu()` 🤷

Reply:
Yeah it has diverged so much I think we should just remove the comment.